controllers/funding/index.js

```diff
@@ -2,101 +2,72 @@
+const { basicContent } = require('../common');
 
 const router = express.Router();
 
+/**
+ * Landing page
+ */
 router.get('/', injectHeroImage('funding-letterbox-new'), async (req, res) => {
-    res.render(path.resolve(__dirname, './views/funding-landing'), {
+    res.render(path.resolve(__dirname, './landing'), {
         title: req.i18n.__('toplevel.funding.title'),
     });
 });
 
-const checkValidPublicationProgramme = (req, res, next) => {
-    const programme = req.params.programme;
-    const validProgrammes = ['a-better-start'];
-    if (!programme || !validProgrammes.includes(programme)) {
-        return res.redirect('/funding');
-    }
-    return next();
-};
+/**
+ * Programmes router
+ * - Programme listings
+ * - Programme detail pages
+ * - Archived programmes
+ */
+router.use('/programmes', require('./programmes'));
 
-router.get(
-    '/publications/:programme?',
-    checkValidPublicationProgramme,
-    injectCopy('insights.documents'),
-    injectHeroImage('a-better-start-new'),
-    async function (req, res, next) {
-        try {
-            const [publicationTags, publications] = await Promise.all([
-                contentApi.getPublicationTags({
-                    locale: req.i18n.getLocale(),
-                    programme: req.params.programme,
-                }),
-                contentApi.getPublications({
-                    locale: req.i18n.getLocale(),
-                    programme: req.params.programme,
-                    searchParams: req.query,
-                }),
-            ]);
+router.use('/strategic-investments', require('./strategic-investments'));
 
-            if (req.params.programme === 'a-better-start') {
-                res.locals.title = 'Publications: A Better Start';
-                res.locals.intro = `We want learning from A Better Start to influence parents, practitioners and policy makers. Read updates from across the A Better Start programme sharing our experiences as we develop innovative ways to give pregnant women and babies the best possible start.`;
-            }
+router.use('/publications', require('./publications'));
 
-            res.render(path.resolve(__dirname, './views/publication-search'), {
-                isBilingual: false, // @TODO: Should we enable Welsh language here?
-                publicationEntries: publications.result,
-                publicationTags: publicationTags,
-                queryParams: req.query,
-                baseUrl: req.baseUrl + req.path,
-                entriesMeta: publications.meta,
-                pagination: publications.pagination,
-                programme: req.params.programme,
-            });
-        } catch (error) {
-            next(error);
-        }
-    }
+router.use('/grants', require('./grants'));
+
+// @TODO: Can this page be migrated to Funding structure in the CMS?
+router.use('/the-big-lunch', basicContent());
+
+// @TODO: Can this section be migrated to Funding structure in the CMS?
```

Comment:
I think most/all of the pages in this section are in Funding and/or out of date, so a big fat YES from me.